### ISSUE TITLE
CSA normalization models are in the wrong `data` subfolder

### ISSUE THREAD

**Issue by joshuacwnewton** [spinalcordtoolbox/spinalcordtoolbox#3558]
Normally, models are installed to the `$SCT_DIR/data` folder (i.e. a `data/` folder in the root of the SCT project folder). However, the models were instead added inside [`$SCT_DIR/spinalcordtoolbox/data`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/tree/master/spinalcordtoolbox/data/) (i.e. an extra folder too deep). 

<details>

This must have slipped by during review of #3515... honestly, an easy thing to miss, given how the paths are displayed in GitHub's file diff:

![Screenshot from 2021-11-01 11-11-20](https://user-images.githubusercontent.com/16181459/139694501-751492b1-e45d-49ee-b66d-da6390408477.png)

^ this is actually `spinalcordtoolbox/spinalcordtoolbox/data`.

</details>

In summary, all that needs to done is move the folder from `${SCT_DIR}/spinalcordtoolbox/data` to `${SCT_DIR}/data`, then update the following link in the codebase:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/138918312d82ae85791a0b26472219b2085a3656/spinalcordtoolbox/scripts/sct_process_segmentation.py#L447-L448

**Comment by jcohenadad** [comment]
wow, good thing you caught that 😅 

but... isn't it CI-tested?

**Comment by joshuacwnewton** [comment]
> but... isn't it CI-tested?

Indeed! Right now, everything functions as expected:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/138918312d82ae85791a0b26472219b2085a3656/spinalcordtoolbox/scripts/sct_process_segmentation.py#L447-L448

It's just an organization issue, rather than a functional issue.
